CMakeLists.txt

```diff
@@ -517,6 +520,27 @@ if(NANOARROW_BUILD_TESTS)
                         gmock_main
                         nanoarrow_coverage_config)
 
+  list(APPEND
```

Comment:
Just kept the loop scoped to adding the define for now, although there is (as you've already mentioned) potential to use this for other parts of the config